_posts/0000-01-02-nachodoritokiwi.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Did you know that world-renowned writer Stephen King was once hit by a car? Just something to consider.